CHANGELOG.md

```diff
@@ -1,159 +1,208 @@
-# Change Log
+# Changelog
 
-## Unreleased
+flake8-pyi uses Calendar Versioning (CalVer).
 
-* **Breaking change:** Previously, flake8-pyi monkey patched flake8's F821
-  (undefined name) check to avoid false positives in stub files. This monkey
-  patch has been removed. Instead, we now recommend to disable F821 when running flake8
-  on stub files.
+## [Unreleased]
```

Comment:
What's the reason for the square brackets here? It looks a bit odd in GitHub's rendering to me:

<img width="2672" height="1426" alt="image" src="https://github.com/user-attachments/assets/daae49fd-7019-4a96-885f-cab973b377a8" />

Reply:
That's supposed to create a link target in some Markdown versions. But since GitHub renders it weirdly and provides link targets automatically, I'll just remove them.